src/wallet-account.ts

```diff
@@ -60,28 +61,36 @@ export class WalletConnection {
     /**
      * Redirects current page to the wallet authentication page.
      * @param contractId The NEAR account where the contract is deployed
-     * @param title Name of the application that will appear as requesting access in Wallet
-     * @param successUrl Optional url to redirect upon success
-     * @param failureUrl Optional url to redirect upon failure
-     * 
+     * @param options An optional options object
+     * @param options.successUrl URL to redirect upon success. Default: current url
```

Comment:
given we are changing this wonder if we should go with https://github.com/near/NEPs/pull/10/files and use one callback URL + return error info from wallet

Reply:
I would prefer to avoid a breaking change if we can. I believe the change proposed here would easily allow adding `callbackUrl` to the `options` later, in support of NEP-10. Right?

Reply:
@chadoh right, didn't mean it to be a breaking change. Not a blocker